src/fmeval/data_loaders/data_sources.py

```diff
@@ -105,3 +106,18 @@ def __reduce__(self):
         """
         serialized_data = (self.uri,)
         return S3DataFile, serialized_data
+
+
+def get_s3_client(uri: str) -> boto3.client:
```

Comment:
Can we move this function to `data_loaders/util.py`? Otherwise, sometime in future, it will create cyclic dependency.

Reply:
That would already create a circular dependency, since we import from `data_loaders/data_sources.py` in `data_loaders/util.py`, and we also use `get_s3_client` here in `data_sources`.